docs/en/latest/config.json

```diff
@@ -96,7 +96,8 @@
             "plugins/fault-injection",
             "plugins/mocking",
             "plugins/degraphql",
-            "plugins/body-transformer"
+            "plugins/body-transformer",
+            "plugins/ai-proxy"
```

Comment:
The recently added `ai-*` plugins should probably all go into a new category. Can be amended in a different PR.